domains/certificates/RegistryConnector/Worker/Program.cs

```diff
@@ -55,6 +69,11 @@
 
     o.AddConsumer<WalletSliceSender>();
 
+    o.AddConsumer<MeasurementEventHandler, MeasurementEventHandlerDefinition>();
+    o.AddConsumer<IssueCertificateNotCompletedConsumer, IssueCertificateNotCompletedConsumerDefinition>();
+
+    o.AddActivitiesFromNamespaceContaining<IssueToRegistryActivity>();
```

Comment:
What does this do?

Reply:
This adds all activities from the namespace that IssueToRegistryActivity is in to masstransit